.github/workflows/manual-upgrade-l1-contracts.yml

```diff
@@ -0,0 +1,84 @@
+name: '[M] Upgrade L1 Contracts'
```

Comment:
how would Moray call this programmatically?

Reply:
@tudor-malene have added a README to the docker upgrade container dir